bin/impl/fetch.sh

```diff
@@ -16,22 +16,46 @@
 
 source "$UNO_HOME"/bin/impl/util.sh
 
-function download_verify() {
+function download_other() {
+  url_prefix=$1
+  tarball=$2
+  expected_hash=$3
```

Comment:
These variables should be defined as local variables, as in:
```bash
local url_prefix; url_prefix=$1
```

Always a good idea to run shellcheck also.